annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -555,7 +571,7 @@ private void validateSubtitleBody (Body body) throws PropertyValidationException
     }
     
     private void validateBodyExists(Body body) throws PropertyValidationException {
-    	if (body == null || body.getValue() == null) {
+    	if (body == null) {
```

Comment:
why was the body.getValue() removed? The meaning og this methos is to prevent empty bodies ...

Reply:
I brought it back. It makes sense. For the two annotations that do not require body value, like LINKING and LINKFORCONTRIBUTING, the validation is in their own methods.